adcc/backends/test_pyscf.py

```diff
@@ -134,6 +143,15 @@ def template_uhf_ch2nh2(self, basis):
         ao_dip = mf.mol.intor_symmetric('int1e_r', comp=3)
         operator_import_test(mf, list(ao_dip))
 
+        # Test magnetic dipole
+        with mf.mol.with_common_orig([0.0, 0.0, 0.0]):
+            ao_magdip = 0.5 * mf.mol.intor('int1e_cg_irxp', comp=3, hermi=2)
+        operator_import_test(mf, list(ao_magdip), "magnetic_dipole")
+
+        # Test linear momentum
+        ao_linmom = -1.0 * mf.mol.intor('int1e_ipovlp', comp=3, hermi=2)
+        operator_import_test(mf, list(ao_linmom), "momentum")
+
```

Comment:
Similar to the psi4 case, this seems repetitive. Might be worth having this as another function similar to the base_test.

Reply:
Done.